exercises/acronym/.meta/src/reference/java/Acronym.java

```diff
@@ -1,24 +1,24 @@
 import java.util.Arrays;
 import java.util.stream.Collectors;
 
-final class Acronym {
+class Acronym {
 
-    private final String acronym;
+    private String acronym;
 
     Acronym(String phrase) {
-        acronym = generateAcronym(phrase);
+        this.acronym = generateAcronym(phrase);
     }
 
-    String get() {
-        return acronym;
+    private String generateAcronym(String phrase) {
+        return Arrays.stream(phrase.split("(?!')\\W"))
+            .filter(word -> !word.isEmpty()) // Remove empty strings from the result of phrase.split
+            .map(word -> word.substring(0, 1)) // Get the first character of each word
+            .collect(Collectors.joining()) // Concatenate the characters
+            .toUpperCase();
     }
 
-    private String generateAcronym(String phrase) {
-        return Arrays.stream(phrase.split("[^a-zA-Z]"))
-                .filter(word -> !word.isEmpty()) // Remove empty strings from the result of phrase.split
-                .map(word -> word.substring(0, 1)) // Get the first character of each word
-                .collect(Collectors.joining()) // Concatenate the characters
-                .toUpperCase();
+    String getAcronym() {
```

Comment:
Why did you change the order of these methods? I would expect this one to come first since it's package private while `generateAcronym` is private 🙂

Reply:
Oh, I'm sorry, I think I mixed up your suggestion from a previous PR. I will change it!

Reply:
No problem, my comment might have been confusing! Thanks! 🙂